lib/msf/ui/console/command_dispatcher/db/klist.rb

```diff
@@ -169,16 +169,17 @@ def set_activation_status(mode, tickets)
 
   # @param [Array<Rex::Socket::RangeWalker>] host_ranges Search for tickets associated with these hosts
   # @param [Array<Integer>, nil] id_search List of ticket IDs to search for
+  # @param [Workspace] workspace to search against
   # @return [Array<Msf::Exploit::Remote::Kerberos::Ticket::Storage::StoredTicket>]
-  def ticket_search(host_ranges, id_search)
+  def ticket_search(host_ranges, id_search, workspace = framework.db.workspace)
```

Comment:
```suggestion
  def ticket_search(host_ranges, id_search, workspace: framework.db.workspace)
```

And updating the yard docs to specify kwargs